app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee

```diff
@@ -108,6 +108,21 @@ $(document).on 'initialize', '#proxy', ->
     tr = $(this).closest("tr")
     tr.find("select,input:not(.destroyer)").removeAttr "disabled"  unless tr.hasClass("deleted")
     false
+
+  # Warning for "catch-all" rules
+  $('td.pattern > input')
+    .keyup (e) ->
+      pattern = e.currentTarget.value
+      isCatchAllMappingRule = pattern in ['', '/']
+      console.log pattern, isCatchAllMappingRule, $(this)
+      $(this)
+        .tipsy(if isCatchAllMappingRule then 'enable' else 'disable')
+        .tipsy(if isCatchAllMappingRule then 'show' else 'hide')
+        .toggleClass 'catch-all-warning', isCatchAllMappingRule
+    .each ->
+      initialPattern = $(this).attr 'value'
+      if initialPattern not in ['', '/']
+        $(this).tipsy 'disable'
```

Comment:
The warning is showing only in the first row.
If you add more mapping rules is not working:
![warnings](https://user-images.githubusercontent.com/13486237/54982132-51fbc580-4faa-11e9-91bd-c4d344d78b9a.png)

Reply:
You may need to include it for manually added proxy rules in line 81: https://github.com/3scale/porta/blob/984e8213494d7561b96e346b78acec98caa079be/app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee#L81